packages/appium/lib/appium.js

```diff
@@ -516,6 +524,17 @@ class AppiumDriver extends BaseDriver {
     // handling the command and which didn't
     plugins.length && this.logPluginHandlerReport({cmd, cmdHandledBy});
```

Comment:
tangential, but maybe `logPluginHandlerReport` should handle "no plugins" instead of `plugins.length &&`

Reply:
yeah reasonable